src/app/pubspec.yaml

```diff
@@ -9,6 +9,9 @@ environment:
 dependencies:
   dio: ^5.4.1
   equatable: ^2.0.5
+  firebase_analytics: ^10.10.4
```

Comment:
Is this necessary?

Reply:
It says in the documentation to add the analytics package to use the full functionalities of remote config. This could be more so for a/b testing and other similar things, but I figured it would be fine to add it. I can try to remove it if you want to see if everything still works.

Reply:
Keep it.